rego/Drive.rego

```diff
@@ -12,13 +12,35 @@ LogEvents := utils.GetEvents("drive_logs")
 #
 # Baseline GWS.DRIVEDOCS.1.1v0.1
 #--
-NonCompliantOUs1_1 contains OU if {
+GetFriendlyValue1_1(Value, AcceptableValues) := "Sharing is Properly Configured" if {
+    Value in AcceptableValues == true
+}
+else := "Sharing Outside Domain is not properly configured."
```

Comment:
Ideally this function would give the user more information, such as "Sharing outside domain is ON" or "Sharing outside domain is set to allowlisted domains." Just saying if it's properly configured or not doesn't really give any more details than the old ReportDetailsBoolean function.